.flake8

```diff
@@ -0,0 +1,12 @@
+[flake8]
```

Comment:
intended to be part of this PR or leftover? I think leftover?

Reply:
Removed